TeamCharles/workforce-management
# 29 training program add view

## Status
**READY

## Documentation

[ X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

## Resolves Issue Number

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
git pull --prune
git checkout <29 and hit `tab`>
dotnet run
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Go to Training Program and Add a training Program.
2. Create a Program and add as many people as you want.
3. Then check the training program index page and watch the new program with the attendee count. 

## Impacted Files in Application

Controllers/TrainingProgramController.cs  /// Views/TrainingProgram/Add.cshtml /// ViewModels/ TrainingProgramAdd.cs
